calendar_caldav/readme/INSTALL.rst

```diff
@@ -0,0 +1,3 @@
+Make sure to install this version of caldav:
+
+`pip3 install git+https://github.com/tarteo/caldav`
```

Comment:
Because of https://github.com/modoboa/caldav/commit/5b5b971dbd75e174ab0376acc6899eec4e78a4e0? Please elaborate

Reply:
Yes, I used it because vobject is really slow.

Reply:
But vanilla one one still works?

Reply:
No, the related PR is blocked because the changes are not backward compatible. https://github.com/python-caldav/caldav/pull/28#issuecomment-379197423

Reply:
That's dangerous :| but I guess you've worked through all available alternatives?

Reply:
Yes, using a branch is fine IMO. We can just wait until master has icalendar support and then change it.

Reply:
master branch of the caldav library now has icalendar support.  Release 0.7 will soon be out.

Reply:
I should probably add, one will still get a vobject if doing some_event.instance - backward compatibility and all that.  However, some_event.icalendar_instance should do the trick.